<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="49.338" tests="3" failures="2">
  <testsuite name="Root Suite" timestamp="2025-06-17T09:06:57" tests="0" file="cypress\e2e\tests\LoginTest.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Success and Fail login flow" timestamp="2025-06-17T09:06:57" tests="3" time="49.322" failures="2">
    <testcase name="Success and Fail login flow should login successfully with valid credentials" time="17.432" classname="should login successfully with valid credentials">
      <failure message="Timed out retrying after 5000ms: expected &apos;https://goodbudget.com/home&apos; to include &apos;/login&apos;" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 5000ms: expected 'https://goodbudget.com/home' to include '/login'
    at LoginPage.loginWithUI (webpack://cypress-framework/./cypress/e2e/pages/LoginPage.js:31:17)
    at Context.eval (webpack://cypress-framework/./cypress/e2e/tests/LoginTest.cy.js:28:13)]]></failure>
    </testcase>
    <testcase name="Success and Fail login flow should fail to login with invalid credentials" time="13.059" classname="should fail to login with invalid credentials">
    </testcase>
    <testcase name="Success and Fail login flow should perform login and logout" time="0.421" classname="should perform login and logout">
      <failure message="Cannot read properties of undefined (reading &apos;should&apos;)" type="TypeError"><![CDATA[TypeError: Cannot read properties of undefined (reading 'should')
    at Context.eval (webpack://cypress-framework/./cypress/e2e/tests/LoginTest.cy.js:50:13)]]></failure>
    </testcase>
  </testsuite>
</testsuites>